core/typedefs.h

```diff
@@ -54,6 +54,11 @@ static_assert(__cplusplus >= 201703L);
 #define _MKSTR(m_x) _STR(m_x)
 #endif
 
+// Allow comma separators within macros.
+#ifndef _COMMA
+#define _COMMA ,
+#endif
+
```

Comment:
Isn't this like my `SEMICOLON` define?

Reply:
It is quite similar, yes. This is, to my knowledge, the only way to pass a comma in a macro without it being treated as a separator for the macro itself. The closest workaround I've found is wrapping the whole thing in parenthesis, but that was inconsistent in my experience. It might not actually be necessary here though, will need to double-check my implementation whenever I get around to rebasing

Reply:
> It might not actually be necessary here though, will need to double-check my implementation whenever I get around to rebasing

If it's needed, it's needed!